config/simulations/README.md

```diff
@@ -9,7 +9,76 @@ files will be in the `output/physiology` directory.
 Note that these are only utilized for the physiology gradle command and are not
 part of the normal Synthea execution procedure.
 
+
+## Configuration  
+
+You will need to set the following two fields to `true` in the `src/main/resources/synthea.properties` file.  
```

Comment:
There are two different ways the physiology models can be invoked and this is conflating them a bit.  (1) The `./gradlew physiology --args="config/simulations/[config name].yml"` command right below this invokes a simulation defined in that file, those simulations point to one of the physiology models. (2) You can also create a module with a Physiology state, run Synthea with it, and when that state gets hit it will run the corresponding model. Only (2) here requires these two settings.

Reply:
Removed.